src/Symfony/Component/Workflow/Workflow.php

```diff
@@ -177,9 +178,19 @@ public function getName()
 
     private function doCan($subject, Marking $marking, Transition $transition)
     {
-        foreach ($transition->getFroms() as $place) {
-            if (!$marking->has($place)) {
-                return false;
+        if ($this->markingStore instanceof UniqueTransitionInputInterface) {
+            // Marking must have ONE place
+            foreach ($transition->getFroms() as $place) {
+                if ($marking->has($place)) {
+                    break;
+                }
+            }
```

Comment:
I think I got over exited when I added the `break`. https://github.com/symfony/symfony/pull/19629/commits/df22253aff68d05014f7f19f7ce666c34b14bb1e